### DISCUSSION TITLE
Spellchecker on Windows 10

### DISCUSSION THREAD

**Discussion by twckr** [KDE/ghostwriter#715] [Q&A]
Hello!

I am using the portable version on Windows 10 but could not get the programm to recognize different hunspell (dic-files) languages is there a certain location in the programm folder these files have to be copied to?

Thomas

P.S. Thanks for this awesome editor!

**Comment by Cigeus** [comment]
I'm running into the same issue. I really like Ghostwriter aside from this.

**Reply by Cigeus** [reply]
Well, I have managed to get a Hunspell dictionary into the data/dictionaries folder, but the program still won't recognise it.

**Reply by Cigeus** [reply]
I finally figured it out. All you need is a .dic file in the data/dictionaries folder of your installation file.
The folder may not exist at first, I think that ghostwriter only makes the data folder after it has things to save.
